recipes/podman/build.sh

```diff
@@ -15,7 +15,6 @@ then
   export CPPFLAGS="${CPPFLAGS} -DHAVE_SETNS"
 fi
 
-sed -i.bak '/exclude_graphdriver_devicemapper/d' "${module_path}/Makefile"
```

Comment:
Upstream's `Makefile` unconditionally sets `exclude_graphdriver_devicemapper`.
I haven't dug into why. I'll open an upstream issue to ask why that is. (They do list it under https://podman.io/getting-started/installation#build-and-run-dependencies though, which is why I packaged `libdevmapper` before.)

Reply:
(It compiles fine with or without `libdevmapper` -- I just haven't made myself familiar with what advantages/disadvantages this implies.)

Reply:
I have simply overlooked a comment in upstream's build docs.
Device-mapper is currently not officially supported, see https://github.com/containers/podman/issues/8262.